me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/RedirectUriReceiverActivity.java

```diff
@@ -27,8 +27,17 @@ public void onError(Throwable throwable) {
   protected void onCreate(@Nullable Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     State currentState = IDmeWebVerify.getCurrentState();
+
     if (currentState == null) {
-      throw new IDmeException("Current state cannot be null");
+      IDmeWebVerify.getInstance().notifyFailure(new IDmeException("Current state cannot be null"));
```

Comment:
well, I think this will not work because anyone could get this message. So this code shouldn't do anything right?